packages/dos-donts-block/src/DoDontItem.tsx

```diff
@@ -263,7 +263,8 @@ export const DoDontItem = React.forwardRef<HTMLDivElement, DoDontItemProps>(
                             </div>
                         )}
                         <div className="tw-w-full tw-flex tw-items-center">
-                            <h3
+                            <span
```

Comment:
Just checking, maybe we should add `aria-label` and `title` for accessibility?

Reply:
Wouldn't use title as it would just help mouse users with a tooltip. 

I don't think this is needed as the shown content would be the same as the `aria-label`, or what do you have in mind to add there? That would be helpful for disabled people. 

[More infos](https://www.aditus.io/aria/aria-label/#over-using-it)